enterprise/kernel/src/main/java/org/neo4j/kernel/impl/store/format/highlimit/HighLimit.java

```diff
@@ -54,7 +54,7 @@
 
     public HighLimit()
     {
-        super( STORE_VERSION, 7, Capability.DENSE_NODES, Capability.SCHEMA, Capability.LUCENE_5 );
+        super( STORE_VERSION, 8, Capability.DENSE_NODES, Capability.SCHEMA, Capability.LUCENE_5 );
```

Comment:
The reason why factory was not introduced here and store version is not changed - is to preserve compatibility between all 3.0 stores and avoid migration completely. Now we do not need migration at all. do we really want to upgrade format in minor release or we can do it in 3.1 only?